smart-contracts/wasm-chain-integration/src/utils.rs

```diff
@@ -252,8 +252,8 @@ pub fn generate_contract_schema_v0(
 }
 
 /// Tries to generate schemas for parameters and return values of methods for a
-/// V1 contract.
-pub fn generate_contract_schema_v1(
+/// V1 contract with a v2 schema.
+pub fn generate_contract_schema_v2(
```

Comment:
I suppose we could have both `V1` and `V2` functions here, since they are both in principle usable.

I guess concordium-std version ultimately determines which version is the correct one. Maybe we should document some compatibilities between cargo-concordium and concordium-std.

Reply:
Where do you propose we document the compatibility? Developer docs?

Reply:
I was thinking just for us in the README to cargo-concordium.